sync/src/net_time_checker.rs

```diff
@@ -141,7 +141,11 @@ impl CKBProtocolHandler for NetTimeProtocol {
             Some(timestamp) => timestamp,
             None => {
                 info!("Peer {} sends us malformed message", peer_index);
-                nc.ban_peer(peer_index, BAD_MESSAGE_BAN_TIME);
+                nc.ban_peer(
+                    peer_index,
+                    BAD_MESSAGE_BAN_TIME,
+                    String::from("send us malformed message"),
```

Comment:
Some are `"send us a malformed message"`, some are `"send us malformed message"` (`send us invalid block`), I think we could use error messages in a same format.

Reply:
Most of them are deserialization failures, some are able to locate that block is illegal.

Ban does not need to locate why it fails, just need to know,  invalid data, or can locate specific data is invalid. Error message doesn't make sense

Reply:
You seems not to have caught what I meant.
Forgive my unclear description that leading your misunderstanding.

1. I mean, you use both `"send us malformed message"` and `"send us a malformed message"` (the difference is the `a`), it's a bit weird.
2. In `"send us malformed message"`, IMHO,  a definite article is required. (also in `"send us invalid block"`, and other messages)